code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm

```diff
@@ -243,14 +243,21 @@ Difficulty: Very Hard
 	return ..()
 
 /obj/item/projectile/colossus
-	name ="death bolt"
-	icon_state= "chronobolt"
+	name = "death bolt"
+	icon_state = "chronobolt"
 	damage = 25
 	armour_penetration = 100
 	speed = 2
 	eyeblur = 0
 	damage_type = BRUTE
 	pass_flags = PASSTABLE
+	movement_type = FLYING | UNSTOPPABLE
+	var/explode_hit_objects = TRUE
+
+/obj/item/projectile/colossus/can_hit_target(atom/target, direct_target = FALSE, ignore_loc = FALSE, cross_failed = FALSE)
+	if(isliving(target))
+		direct_target = TRUE
```

Comment:
```suggestion
	if(isliving(target))
		var/mob/living/L = target
		if(L.stat != DEAD)
			direct_target = TRUE
```

Reply:
idk if that will work, direct target just makes it hit them if the projectile would usually ignore them
I think it needs a thing in on_hit that says if(target is dead) return BULLET_ACT_FORCE_PIERCE or whatever the define is